data/distro.yml

```diff
@@ -605,3 +605,27 @@
     </ol>
 
 
+- name: "Pisi GNU/Linux"
+  logo: "pisi.svg"
+  info:  >
+    <ol class="distrotut">
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>A flatpak package is available in Pisi 2.1 and newer. To install it, run the following as root:</p>
+        <pre><code>
+          <span class="unselectable">#</span> sudo pisi it flatpak
+        </code></pre>
+      </li>
+      <li>
+        <h2>Add the Flathub repository</h2>
+        <p>Flathub is the best place to get Flatpak apps. To enable it, run:</p>
+        <pre><code>
+          <span class="unselectable">#</span> sudo flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
```

Comment:
Same as the command above + please remove `sudo`, it should not be needed and is not recommended for Flatpak commands (Polkit is used to elevate their rights).

Reply:
On Pisi GNU/Linux users need to superuser rights to writing to main file system. Therefore they must to use "sudo" command before installating/removing an app.

Reply:
So Polkit is not supported there?

Reply:
Let me know what is the state of Polkit support on your distro. If elevating rights using Polkit it really is not supported, I think that using sudo for the flatpak command makes sense. I just hope it won't cause any other issues.

Reply:
In fact, we also use Polkit. But Pisi GNU/Linux uses own base system and on terminal it request sudo command.